### ISSUE TITLE
Netflix is no longer available on this device. Please visit netflix.com/compatibledevices for a list of supported devices. (R17)

### ISSUE THREAD

**Issue by banyartibi** [aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#37]
Dear @aidanmacgregor 

I am using [Aidan's TV Netflix v3 (L3 Uncertified) (2.0.build.295) (Jan 2023 - CURRENT)_(Android 6.0+).apk](https://androidfilehost.com/?fid=4279422670115736288)

Application started to push notification at each start that it will not work anymore after 15th next month.

**Comment by reinaldoaru** [comment]
Here getting the same message...

**Comment by tekken56k** [comment]
Same here

**Comment by Webbeh** [comment]
Same here as well.

**Comment by itsskyballs** [comment]
i didnt know why i got an error,but i knew that they ditched it

**Comment by peppemas1** [comment]
same problem android tv box with allwinner h313 this version dont work (Aidan's TV Netflix v3 (L3 Uncertified) (2.0.build.295) (Jan 2023 - CURRENT)_(Android 6.0+) after 15 july, i dont like this netflix policy, how is possible to install a version without check for certified device?

**Comment by Rleenman** [comment]
Same here with meecool km6 😭
I hope somebody fix a new mod.

**Comment by banyartibi** [comment]
Dear @aidanmacgregor is it possible to fix this anyhow?

**Comment by Sinaloense** [comment]
Same here

**Comment by itsskyballs** [comment]
In `BaseESNProvider.smali` line 1885
You can change the string id to a different string so it should work as expected
Heres the file so you can modify it
https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/blob/main/APKtool/com.netflix.ninja-2.0_build_295-295-minAPI23/smali/com/netflix/mediaclient/service/configuration/esn/BaseEsnProvider.smali

**Comment by reinaldoaru** [comment]
> In `BaseESNProvider.smali` line 1885 You can change the string id to a different string so it should work as expected Heres the file so you can modify it https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/blob/main/APKtool/com.netflix.ninja-2.0_build_295-295-minAPI23/smali/com/netflix/mediaclient/service/configuration/esn/BaseEsnProvider.smali

Thanks for your reply. Any value for the string will work?

**Comment by itsskyballs** [comment]
> > In `BaseESNProvider.smali` line 1885 You can change the string id to a different string so it should work as expected Heres the file so you can modify it https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/blob/main/APKtool/com.netflix.ninja-2.0_build_295-295-minAPI23/smali/com/netflix/mediaclient/service/configuration/esn/BaseEsnProvider.smali
> 
> Thanks for your reply. Any value for the string will work?

No,you may decompile the official Netflix app for atv and see the string at line 1885

**Comment by wangxinqing** [comment]
> > > In `BaseESNProvider.smali` line 1885 You can change the string id to a different string so it should work as expected Heres the file so you can modify it https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/blob/main/APKtool/com.netflix.ninja-2.0_build_295-295-minAPI23/smali/com/netflix/mediaclient/service/configuration/esn/BaseEsnProvider.smali
> > 
> > 
> > Thanks for your reply. Any value for the string will work?
> 
> No,you may decompile the official Netflix app for atv and see the string at line 1885

Changing the esn should not work. Netflix should have judged the signature of the apk, and if the signature changes, it will be considered illegal.

**Comment by manleong** [comment]
> > > > In `BaseESNProvider.smali` line 1885 You can change the string id to a different string so it should work as expected Heres the file so you can modify it https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/blob/main/APKtool/com.netflix.ninja-2.0_build_295-295-minAPI23/smali/com/netflix/mediaclient/service/configuration/esn/BaseEsnProvider.smali
> > > 
> > > 
> > > Thanks for your reply. Any value for the string will work?
> > 
> > 
> > No,you may decompile the official Netflix app for atv and see the string at line 1885
> 
> Changing the esn should not work. Netflix should have judged the signature of the apk, and if the signature changes, it will be considered illegal.

i do manage to decompile and get a legit ESN
also, after the modded version of tv app is signed and install to the tv box, it keep prompt me that no internet connection (Sorry We Cannot Reach The Netflix Service (-9))
not sure is it due to the illegal signature

**Comment by shyxun** [comment]
> > > In `BaseESNProvider.smali` line 1885 You can change the string id to a different string so it should work as expected Heres the file so you can modify it https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/blob/main/APKtool/com.netflix.ninja-2.0_build_295-295-minAPI23/smali/com/netflix/mediaclient/service/configuration/esn/BaseEsnProvider.smali
> > 
> > 
> > Thanks for your reply. Any value for the string will work?
> 
> No,you may decompile the official Netflix app for atv and see the string at line 1885

Can you DM me the ESN you used?

**Comment by soymaiku** [comment]
> In `BaseESNProvider.smali` line 1885 You can change the string id to a different string so it should work as expected Heres the file so you can modify it https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/blob/main/APKtool/com.netflix.ninja-2.0_build_295-295-minAPI23/smali/com/netflix/mediaclient/service/configuration/esn/BaseEsnProvider.smali

can i dm you what ESN you used and how?

**Comment by itsskyballs** [comment]
Aidan isnt online years ago

**Comment by itsskyballs** [comment]
> > > > > In `BaseESNProvider.smali` line 1885 You can change the string id to a different string so it should work as expected Heres the file so you can modify it https://github.com/aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod/blob/main/APKtool/com.netflix.ninja-2.0_build_295-295-minAPI23/smali/com/netflix/mediaclient/service/configuration/esn/BaseEsnProvider.smali
> > > > 
> > > > 
> > > > Thanks for your reply. Any value for the string will work?
> > > 
> > > 
> > > No,you may decompile the official Netflix app for atv and see the string at line 1885
> > 
> > Changing the esn should not work. Netflix should have judged the signature of the apk, and if the signature changes, it will be considered illegal.
> 
> i do manage to decompile and get a legit ESN
> also, after the modded version of tv app is signed and install to the tv box, it keep prompt me that no internet connection (Sorry We Cannot Reach The Netflix Service (-9))
> not sure is it due to the illegal signature

Its up to you
Illegal means that the app will not work due to mismatched compile signature

**Comment by jesulonimii** [comment]
@itsskyballs 
Have you gotten yours to work recently?

**Comment by vEditzzz6** [comment]
I am interested in this. I know one guy who has a fully working android mobile mod apk but he will not give me the method but as ive done some sniffing, I find out that the he also has spoofed the app with also ESN spoofed., version and other device info... if anyone is interested, I have the apk myself. But it is protected with jiagu 360....

**Comment by ugarden652-hue** [comment]
@vEditzzz6 
I want it plss

**Comment by hideandseekstheorder** [comment]
please share 🙏🏻
